.gitlab/packaging/deb.yml

```diff
@@ -117,4 +117,94 @@ installer_deb-arm64:
     DD_PROJECT: "installer"
     RELEASE_VERSION: $RELEASE_VERSION_7
     AGENT_MAJOR_VERSION: 7
+    DD_PROJECT: "installer"
+    RELEASE_VERSION: $RELEASE_VERSION_7
+    AGENT_MAJOR_VERSION: 7
```

Comment:
Duplicated from the lines above?
```suggestion
```

Reply:
Indeed, thanks for noticing!
I fixed it in the IOT agent PR on which this PR is based